src/Compilers/VisualBasic/Portable/Symbols/NamedTypeSymbol.vb

```diff
@@ -1187,8 +1187,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Private ReadOnly Property INamedTypeSymbol_IsImplicitClass As Boolean Implements INamedTypeSymbol.IsImplicitClass
             Get
-                ' TODO (tomat):
```

Comment:
@tmat This _seemed_ like a straight forward enough change. Any non-obvious implications?

Reply:
@dotnet/roslyn-compiler PTAL

Reply:
LGTM